platform-service/pyproject.toml

```
@@ -13,7 +13,7 @@ dependencies = [
     "redis~=5.2.1",
     "cryptography>=41.0.7",
     "requests>=2.31.0",
-    "unstract-sdk~=0.55.0rc2",
+    "unstract-sdk~=0.55.0rc3",
```

@gaya3-zipstack Here I could see a discrepancy with the versions, it is 0.55.0rc3 instead of 0.56.0rc3.
```suggestion
    "unstract-sdk~=0.56.0rc3",
```

---

Great catch @pk-zipstack Thank you!